functions/go/apply-setters/applysetters/apply_setters_test.go

```diff
@@ -390,11 +390,43 @@ data:
 			expectedResources: `apiVersion: v1
 kind: Service
 metadata:
-  name: # kpt-set: ${app}
-  namespace: # kpt-set: ${ns}
+  name: null # kpt-set: ${app}
+  namespace: null # kpt-set: ${ns}
+image: :1.7.1 # kpt-set: ${image}:${tag}
+env: # kpt-set: ${env}
+  - null
+`,
+		},
+		{
+			name: "set non-empty values from empty values",
+			input: `apiVersion: v1
+kind: Service
+metadata:
+  name: null # kpt-set: ${app}
+  namespace: null # kpt-set: ${ns}
 image: :1.7.1 # kpt-set: ${image}:${tag}
 env: # kpt-set: ${env}
   - null
```

Comment:
As a user of this function this would be **very** unexpected for me. Providing null for an array value and having a one element array with a null element set seems strange.

Reply:
Updated to FlowStyle empty sequence.